cmd/create/cluster/spec_test.go

```diff
@@ -50,7 +50,7 @@ var _ = Describe("Cluster Configuration Creation", func() {
 	) {
 		out, err := clusterConfigFor(r.Reporter, in, creator, aws)
 		Expect(err).To(BeNil())
-		Expect(out).To(test.MatchExpected(expected, cmpopts.IgnoreUnexported(cmv1.ExternalAuthConfig{})))
+		Expect(out).To(matchers.MatchExpected(expected, cmpopts.IgnoreUnexported(cmv1.ExternalAuthConfig{})))
```

Comment:
I had to move this helper from the `pkg/test` package since that imports `pkg/aws` and would otherwise have created an import cycle.